style.css

```diff
@@ -571,18 +583,604 @@ footer h6 {
   padding-top: 12px;
 }
 
+#list-of-project2-img {
+  display: none;
+}
+
+#list-of-project2-imgb {
+  display: none;
+}
+
+#desktop-img1a {
+  display: none;
+}
+
+#mobile-img {
+  display: block;
+}
+
 #resume-img {
   display: flex;
   padding-top: 10px;
 }
 
+#btn-imgA {
+  display: none;
+}
+
 #img-geo {
   display: none;
 }
 
 #cross-img {
+  display: block;
   width: 20px;
   height: 20px;
   margin-left: 24px;
   margin-top: 10px;
 }
+
+/* media query */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    height: 100vh;
+    overflow-x: hidden;
+  }
+
+  #navbar-img {
+    content: '';
+    display: none;
+  }
+
+  #nav-list1 {
+    margin: 2px 30%;
+    padding: 0 20px;
+  }
+
+  #nav-list2 {
+    margin: 2px 30%;
+    padding: 0 20px;
+  }
+
+  #nav-list3 {
+    margin: 2px 30%;
+    padding: 0 20px;
+  }
+
+  #nav-list4 {
+    margin: 2px 30%;
+    padding: 0 20px;
+  }
+
+  #header-section nav .navbar-list-items li a {
+    display: inline;
+    text-decoration: none;
+    color: #1a2236;
+    flex-direction: row;
+    margin-right: 0;
+    width: 100%;
+    height: 20px;
+    justify-content: space-around;
+    align-items: center;
+    transform: rotateX(45deg);
+  }
+
+  #header-section {
+    display: block;
+    width: 100%;
+  }
+
+  #header-section #navbar-container nav ul {
+    display: flex;
+    width: 100%;
+    list-style-type: none;
+    text-decoration: none;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    position: absolute;
+    top: 0;
+    height: 104px;
+    padding: 10px;
+  }
+
+  #left-float {
+    display: none;
+  }
+
+  #section-main h1,
+  p {
+    content: '';
+    width: 40%;
+    height: 350px;
+    padding: 1em;
+    float: left;
+    margin-right: 0;
+    margin-top: 0;
+    font-size: 54px;
+    margin-left: 2%;
+  }
+
```

Comment:
- [ ] Kindly fix the responsiveness of your header/headline. Your text below your title in the header is overlapping the button below it in some screen sizes.
![image](https://user-images.githubusercontent.com/61919617/172934510-c5873936-ac40-4647-8c76-f60d4be1e676.png)

Reply:
Done